consensus/metrics.go

```diff
@@ -3,11 +3,12 @@ package consensus
 import (
 	"github.com/go-kit/kit/metrics"
 	"github.com/go-kit/kit/metrics/discard"
-
```

Comment:
This newline is meaningful, we group by: `stdlib`, `external packages`, `internal packages`.